mne/evoked.py

```diff
@@ -921,8 +929,31 @@ def combine_evoked(all_evoked, weights):
     evoked.info['bads'] = bads
     evoked.data = sum(w * e.data for w, e in zip(weights, all_evoked))
     evoked.nave = new_nave
-    evoked.comment = ' + '.join(f'{w:0.3f} × {e.comment or "unknown"}'
-                                for w, e in zip(weights, all_evoked))
+
+    comment = ''
+    for idx, (w, e) in enumerate(zip(weights, all_evoked)):
+        if np.isclose(w, 1.):
+            if idx != 0:  # add no "+" to leading term
+                comment += '+ '
+        elif np.isclose(w, -1.):
+            comment += '-'
+            if idx != 0:  # add space before first term
```

Comment:
maybe this should go. Idea is that it produces:

```
–(x) – (y)
```
instead of
```
– (x) – (y)
```
but maybe the latter case is more consistent.